static/js/hotkey.js

```diff
@@ -495,6 +496,11 @@ exports.process_hotkey = function (e, hotkey) {
         return false;
     }
 
+    if (event_name === "narrow_compose" && compose_state.composing()) {
+        narrow.to_compose_target();
+        return true;
+    }
+
```

Comment:
This spot in the `hotkey.js` sequence doesn't seem super well-thought through.  Let's move it to just before the `processing_text` one?